main/isotp_ble_bridge_main.c

```diff
@@ -279,15 +366,26 @@ void app_main(void)
     ble_server_callbacks callbacks = {
         .data_received = received_from_ble,
         .notifications_subscribed = notifications_enabled,
-        .notifications_unsubscribed = notifications_disabled};
+        .notifications_unsubscribed = notifications_disabled
+    };
     ble_server_setup(callbacks);
 
     // Setup WiFi server
-    wifi_server_setup();
+    // wifi_ap_server_setup(); we host an SSID and clients connect to us
```

Comment:
probably a better way to do this with some kind of CONFIG flag? honestly it's not even really needed anymore. if your website is `HTTPS`, you'll never be able to call an `HTTP (ws)` websocket from it. it would need to be `wss://` and there's no way to issue a valid cert for a "local" IP + hostname combo that a browser would trust without extra work. not worth it